src/linearmixedmodel.jl

```diff
@@ -356,53 +380,44 @@ StatsBase.fitted(m::LinearMixedModel{T}) where {T} = fitted!(Vector{T}(undef, no
 """
     fixef!(v::Vector{T}, m::LinearMixedModel{T})
 
-Overwrite `v` with the pivoted and, possibly, truncated fixed-effects coefficients of model `m`
-"""
-fixef!(v::AbstractVector{T}, m::LinearMixedModel{T}) where {T} =
-    ldiv!(feL(m)', copyto!(v, m.L.blocks[end, end-1]))
+Overwrite `v` with the pivoted fixed-effects coefficients of model `m`
 
+For full rank models the length of `v` must be the rank of `X`.  For rank-deficient models
```

Comment:
"full-rank" for consistency with "rank-deficient"

Reply:
Thanks for catching that.  I will fix it.